server/repositories/cardRepository.js

```diff
@@ -0,0 +1,43 @@
+const _ = require('underscore');
+
+const logger = require('../log.js');
+const BaseRepository = require('./baseRepository.js');
+
+class DeckRepository extends BaseRepository {
+    getCards(brief, callback) {
```

Comment:
This is probably very nitpicky, but I feel like this would be better served as two separate methods rather than passing a boolean flag whose meaning is unclear unless you look at the method itself.

(spoilers, I'm probably going to separate `getStrength(bool)` into `getPrintedStrength()` and `getStrength()` in the near future)